TF_Service_dotNet/TouchFree/ServiceTypes.cs

```diff
@@ -180,6 +182,22 @@ public static class Utilities
 
             return returnVal;
         }
+
+        public static System.Numerics.Vector3 LeapToNumerics(Leap.Vector _leap)
```

Comment:
Should this specify in the method name that is produces a Vector3?

Reply:
My thought was that (if need be) we could overload this method to handle Vector2s as well so you can just call `LeapToNumerics` for the relevant type without having to specify the type explicitly in the function name.

Reply:
I think that makes sense though I think for clarity it should be LeapToNumericsVector in that case then? I guess it's personal preference but I would prefer the name not seem so broad, 'Leap' covers a lot of types and so does 'Numerics'